.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,34 +1,32 @@
-Please pick a meaningful title for your pull request using sentence case.
+Instructions:
 
-Do not overuse punctuation in the title like `(chore):`. If it is helpful feel free to start with a project name, though, like `ProjectX: Implement some feature`.
+- Pick a meaningful title for your pull request. (Use sentence case.)
+  - Prefix the title with an emoji to identify what is being done. (Copy-paste from the list below.)
+  - Do not overuse punctuation in the title (like `(chore):`).
+  - If it is helpful, use a simple prefix (like `ProjectX: Implement some feature`).
+- Enter a succinct description that says why the PR is necessary, and what it does.
+  - Mention the GitHub issue that is being addressed by the pull request.
+  - The keywords `Fixes`, `Closes`, or `Resolves` followed the issue number will automatically close the issue.
 
-# Title instructions above.
-
-Enter a succinct description of what is achieved by the PR. Ideally describe why the change is being made.
-
-Bullet points like
+Example of a good description:
 
 - Implements aspect X
 - Leaves out feature Y because of A
 - Improves performance by B
 - Improves accessibility by C
 
-really help with making this more readable.
-
-Fixes/Closes/Related-to #1 (enter issue number, except in rare cases where none exists).
-
-It is also helpful to add an emoji before the commit message to identify the kind of work done on a single commit. See the following suggestions below:
-
-- 🐛 - `:bug:` Bug fix
-- ✨ - `:sparkles:` New feature
-- ✅ - `:white_check_mark:` Tests
-- 🔥 - `:fire:` P0
-- 🚀 - `:rocket:` Performance improvements
-- 🖍️ - `:crayon:` CSS / Styling
-- ♿ - `:wheelchair:` Accessibility
-- 🌐 - `:globe_with_meridians:` i18n (Internationalization)
-- 📖 - `:book:` Documentation
-- ♻️ - `:recycle:` Refactoring (like moving around code w/o any changes)
-- 🏗️ - `:building_construction:` Infrastructure / Tooling / Builds / CI
-- ⏪ - `:rewind:` Revert
-- 🚮 - `:put_litter_in_its_place:` Deleting code
+Emojis for categorizing pull requests:
+
+✨ New feature (`:sparkles:`)
```

Comment:
This should be a list.

Reply:
It is a list when displayed in raw text. Adding a dash to each line makes it very cumbersome to copy-paste the right emoji. See link in this PR description for what it will look like inside the compose window of a PR.

Reply:
Use a double space at the end of each line to insert a `<br>`, then.

Source:
```text
this  
is  
also  
pretty  
printed  
in  
source  
text  
```

Rendered:
```md
this  
is  
also  
pretty  
printed  
in  
source  
text  
```

Reply:
Nice. Done in #13570